README.md

```diff
@@ -0,0 +1,13 @@
+# Gmp-detector
+
+This repository's aim to contain a reusable script, to detect all GMP function usages in different repositories.
+
+In order to do that we create a simple script, which lists all the functions in a specified folder.
+
+## Usage
+
+You may need to specify the path to the folder in the first prameter, use the following command:
+
+~~~~bash
+./mpz_search.sh [path]
```

Comment:
For me, it's not evident whether the `path` argument is required or not, and the behavior when it's missing. What do you think, can we make this usage description a bit more exact?

Reply:
Sure, I will describe it more exact. :smile: